packages/website/docs/createQuerySuggestionsPlugin.md

```diff
@@ -127,3 +157,36 @@ const querySuggestionsPlugin = createQuerySuggestionsPlugin({
   },
 });
 ```
+
+### `categoryAttribute`
+
+> `string | string[]`
+
+The attribute or attribute path to display categories for.
+
+#### Example
+
+```js
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'instant_search_demo_query_suggestions',
+  categoryAttribute: [
+    'instant_search',
+    'facets',
+    'exact_matches',
+    'hierarchicalCategories.lvl0',
+  ],
+});
+```
+
+### `categoriesLimit`
+
+> `number`
+
+The number of items to display categories for.
```

Comment:
I wrote this description myself quickly but not sure it's very comprehensible. Wdyt?

Reply:
Can you quickly describe it to make sure it's clear to me?

Reply:
(With `categoriesPerItem: 2`)

### `categoriesLimit: 1`

![image](https://user-images.githubusercontent.com/6137112/109517699-d4039b80-7aa9-11eb-9b0a-de1deb5e7be4.png)

### `categoriesLimit: 3`

![image](https://user-images.githubusercontent.com/6137112/109516455-820e4600-7aa8-11eb-8cad-3040ecdafbde.png)

Reply:
Got it! It's how many items get categories. I'm wondering if the name of the parameter is optimal though. Now that I know what it is, `categoriesLimit` still doesn't convey that idea. Have you thought of something more descriptive like `itemsWithCategories`? It doesn't convey as much that it expects a `number` but it's clearer.

Reply:
Yeah I'm not convinced by the name either.

How about those?

- `numberOfItemsWithCategories`
- `numberOfCategoriesPerItem`

Reply:
I find `numberOf` superfluous, the type conveys this information already.

- `itemsWithCategories`
- `categoriesPerItem`

This is simpler.

Reply:
OK—I'll take care of that in another PR.